src/lambda/lambda.rs

```diff
@@ -1,128 +1,137 @@
-use super::expression::{Context, Expression, Operation};
+use super::expression::Operation;
 use crate::request_template::RequestTemplate;
 
 #[derive(Clone, Debug)]
 pub struct Lambda<A> {
-  _output: PhantomData<fn() -> A>,
-  pub expression: Expression,
+    _output: PhantomData<fn() -> A>,
+    pub expression: expression::Expression,
 }
 
 impl<A> Lambda<A> {
-  fn box_expr(self) -> Box<Expression> {
-    Box::new(self.expression)
-  }
-  pub fn new(expression: Expression) -> Self {
-    Self { _output: PhantomData, expression }
-  }
-
-  pub fn eq(self, other: Self) -> Lambda<bool> {
-    Lambda::new(Expression::EqualTo(self.box_expr(), Box::new(other.expression)))
-  }
-
-  pub fn to_unsafe_js(self, script: String) -> Lambda<serde_json::Value> {
-    Lambda::new(Expression::Unsafe(Operation::JS(self.box_expr(), script)))
-  }
-
-  pub fn to_input_path(self, path: Vec<String>) -> Lambda<serde_json::Value> {
-    Lambda::new(Expression::Input(self.box_expr(), path))
-  }
+    fn box_expr(self) -> expression::Expression {
+        self.expression
+    }
+
+    pub fn new(expression: expression::Expression) -> Self {
+        Self {
+            _output: PhantomData,
+            expression,
+        }
+    }
+
+    pub fn eq(self, other: Self) -> Lambda<bool> {
+        Lambda::new(expression::Expression::EqualTo(
+            Box::new(self.box_expr()),
+            Box::new(other.expression),
+        ))
+    }
+
+    pub fn to_unsafe_js(self, script: String) -> Lambda<serde_json::Value> {
+        Lambda::new(expression::Expression::Unsafe(Operation::JS(
+            Box::new(self.box_expr()),
+            script,
+        )))
+    }
+
+    pub fn to_input_path(self, path: Vec<String>) -> Lambda<serde_json::Value> {
+        Lambda::new(expression::Expression::Input(Box::new(self.box_expr()), path))
+    }
 }
 
 impl Lambda<serde_json::Value> {
-  pub fn context() -> Self {
-    Lambda::new(Expression::Context(expression::Context::Value))
-  }
+    pub fn context() -> Self {
+        Lambda::new(expression::Expression::Context(expression::Context::Value))
+    }
 
-  pub fn context_field(name: String) -> Self {
-    Lambda::new(Expression::Context(Context::Path(vec![name])))
-  }
+    pub fn context_field(name: String) -> Self {
+        Lambda::new(expression::Expression::Context(expression::Context::Path(vec![name])))
+    }
 
-  pub fn context_path(path: Vec<String>) -> Self {
-    Lambda::new(Expression::Context(Context::Path(path)))
-  }
+    pub fn context_path(path: Vec<String>) -> Self {
+        Lambda::new(expression::Expression::Context(expression::Context::Path(path)))
+    }
 
-  pub fn from_request_template(req_template: RequestTemplate) -> Lambda<serde_json::Value> {
-    Lambda::new(Expression::Unsafe(Operation::Endpoint(req_template)))
-  }
+    pub fn from_request_template(req_template: RequestTemplate) -> Lambda<serde_json::Value> {
+        Lambda::new(expression::Expression::Unsafe(Operation::Endpoint(req_template)))
```

Comment:
We can drop the lambda functions that were referencing the deprecated expressions.